src/components/AbuseReportForm/AbuseReportForm.tsx

```diff
@@ -1,31 +1,46 @@
-import React, { useState } from "react";
+import React, { useState, useEffect, useRef } from "react";
 import { css } from "emotion";
+import { useForm } from "react-hook-form";
+
+import { palette } from "@guardian/src-foundations";
 import { textSans } from "@guardian/src-foundations/typography";
 import { space, neutral } from "@guardian/src-foundations";
 import { Button } from "@guardian/src-button";
 import { SvgClose } from "@guardian/src-svgs";
 
+import { Pillar } from "../../types";
 import { reportAbuse } from "../../lib/api";
 
 type Props = {
   commentId: number;
+  pillar: Pillar;
 };
 
-const validate = ({ categoryId }: { categoryId: number }) =>
-  !categoryId ? { category: errors.category } : null;
-
-const errors = {
-  category: "Please select a category"
+type formData = {
+  categoryId: number;
+  reason?: string;
+  email?: string;
 };
 
 const formWrapper = css`
+  z-index: 1;
+  border: 1px solid #dcdcdc;
   position: absolute;
   width: 300px;
+  top: 0;
+  right: 0;
+  display: flex;
+  flex-direction: column;
   padding: ${space[3]}px;
   background-color: white;
   ${textSans.xsmall()};
 `;
 
+const labelStyles = (pillar: Pillar) => css`
+  color: ${palette[pillar][400]};
```

Comment:
What's 400? Is this something new? I thought pillar colours were named.

Reply:
it is used elsewhere, and would make the design look constant. I think it is safe to keep. We can reopen this later when we run through final design